app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -328,6 +328,8 @@ def save_container_component_statuses_inventory(ems, hashes, target = nil)
   end
 
   def save_container_inventory(container_definition, hash, _target = nil)
+    return if hash.nil?
+    
     # The hash could be nil when the container is in transition (still downloading
```

Comment:
@imtayadeway no, actually I think that this PR would break what is mentioned in the comment.